packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -56,7 +56,9 @@ export function createTokenIntrospectionMiddleware({
       )
       let tokenInfo: TokenInfo
       try {
-        tokenInfo = await tokenIntrospectionClient.introspect(token)
+        tokenInfo = await tokenIntrospectionClient.introspect({
```

Comment:
Let's add the TokenIntrospection `Client` interface to `interface AppServices`, so we have proper types when resolving this dependency

Reply:
:+1: Thanks
You've resolved my confusion https://github.com/interledger/rafiki/issues/1002#issuecomment-1397260511

Reply:
I wish `container.singleton()` automagically did that already